ostelco-ios-client/TestAppDelegate.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  TestAppDelegate.swift
+//  ostelco-ios-client
+//
+//  Created by Ellen Shapiro on 4/8/19.
+//  Copyright © 2019 mac. All rights reserved.
+//
+
+import UIKit
+
+class TestAppDelegate: UIResponder, UIApplicationDelegate {
```

Comment:
All this goop is based on this technique: https://qualitycoding.org/ios-app-delegate-testing/